content/en/continuous_integration/setup_pipelines/buildkite.md

```diff
@@ -33,6 +33,33 @@ The [Pipelines][3] and [Pipeline Executions][4] pages populate with data after t
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+## Setting custom tags
```

Comment:
All the blocks above are at the `##` and I don't think this makes a lot of sense as a subtitle of `## Visualize pipeline data in Datadog` or does it?

Reply:
it would not substitute the next section, just come first and inside `Configure the Datadog integration`. Anyway if Jenkins already does this... it might be better to move the Jenkins section too. Whatever you decide it's best